### PR TITLE
Styled pages and buttons

### PR DESCRIPTION
**Pull request by rshamoon**
Co-authored-by: njrinker <njrinker@users.noreply.github.com>

### CODE REVIEW

**Review by ivanOzerets — APPROVED** [review]

File: source/HTML/daily.html [hunk 1]
```diff
@@ -34,8 +34,8 @@
                 -->
                 <div class="icons-for-daily-and-monthly"> 
                     <!-- Icons for menu and zoom out -->
-                    <i id="menu" class="fas fa-bars fa-3x"></i>
-                    <i id="zoom-out" class="fas fa-search-minus fa-3x"></i>
+                    <button class="exterior-buttons" id="menu-button"><i class="button-left button-top fas fa-bars icon-size"></i></button>
+                    <button class="exterior-buttons" id="zoom-out-button"><i class="button-right button-top fas fa-search-minus icon-size"></i></button>
                     
                     <!--
                         TODO: Which icons did we want to use???????????
```
Comment (maniacalhamster) [inline source/HTML/daily.html:41]: I think buttons were decided in this commit right?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (maniacalhamster) [inline .vscode/settings.json:2]: Was this added accidentally :0 ?

**Review by maniacalhamster — CHANGES_REQUESTED** [review]
Just noticed 2 things:
- [ ] settings.json for vscode got added in (probably by mistake?)
- [ ] TODO comment about choosing a style for buttons is still a part of the code

Other than that the code looks great!

**Review by asya-balas — COMMENTED** [review]
Also do we want to do display changing here in the css by having a class being added to the body element in the html (I believe similar to how we have the settings class in 
Lab 7 that is added and removed depending on the state) or will that be handled elsewhere in the js?

File: .vscode/settings.json (hunk 2 above)
Comment (rshamoon) [reply .vscode/settings.json]: good catch, I believe it should be fixed now

**Review by maniacalhamster — APPROVED** [review]

**Review by asya-balas — APPROVED** [review]

**Pull request merged**